### ISSUE TITLE
libav compatibility issue with av_log_format_line

### ISSUE THREAD

**Issue by julietteTaka** [mikrosimage/avTranscoder#117]
```
/home/juliette/Programmation_compilation/webOpenOFX/TuttleOFX/plugins/image/io/AudioVideo/avTranscoder/src/AvTranscoder/log.cpp:12:75: error: ‘av_log_format_line’ was not declared in this scope
  av_log_format_line(ptr, level, fmt, vl, line, sizeof(line), &print_prefix);
                                                                           ^
```

```
/usr/include/libavutil/version.h:39:#define LIBAVUTIL_VERSION_MAJOR 52
/usr/include/libavutil/version.h:40:#define LIBAVUTIL_VERSION_MINOR  3
/usr/include/libavutil/version.h:41:#define LIBAVUTIL_VERSION_MICRO  0
```

**Comment by julietteTaka** [comment]
```
/home/juliette/Programmation_compilation/webOpenOFX/TuttleOFX/plugins/image/io/AudioVideo/avTranscoder/src/AvTranscoder/transform/AudioTransform.cpp:16:39: fatal error: libswresample/swresample.h: Aucun fichier ou dossier de ce type
  #include <libswresample/swresample.h>
                                       ^
```

**Comment by cchampet** [comment]
I've just found that libav never had function ```av_log_format_line```...

**Comment by fabiencastan** [comment]
Yes, it's only in ffmpeg...

**Comment by cchampet** [comment]
Can you test it?

**Comment by fabiencastan** [comment]
Works fine.

**Comment by cchampet** [comment]
Fixed with #127
